commands/ci/migrate-manifest.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs')
+const cli = require('heroku-cli-util')
+const co = require('co')
+const BB = require('bluebird')
+const writeFile = BB.promisify(fs.writeFile)
+const unlinkFile = BB.promisify(fs.unlink)
+
+function* run (context, heroku) {
+  const appJSONPath = `${process.cwd()}/app.json`
+  const appCiJSONPath = `${process.cwd()}/app-ci.json`
+
+  let appJSON, appCiJSON
+
+  try {
+    appJSON = require(appJSONPath)
+  } catch (e) {
+    appJSON = {}
+  }
+
+  try {
+    appCiJSON = require(appCiJSONPath)
+  } catch (e) {
+    throw new Error(`Couldn't load app-ci.json. This command must be run from a directory containing an app-ci.json file`)
```

Comment:
This could probably be something more friendly. "This codebase isn't using app-ci.json, so you are good to go!"